docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md

```diff
@@ -0,0 +1,107 @@
+    5) Press *Win + R* again, type and enter ***msiexec /regserver*** and hit *Enter*
+
+- Acquire full permissions on the drive for installation. It is possible that the error 1603 occurs because you don’t have full permissions on the file location. It's also not as easy as some of the other solutions:
+
+    1) Open *File Explorer*, right click on the drive containing the installation location and select *Properties*.
+    2) Go to *Security* tab and click on *Edit*.
+    3) Single click on *SYSTEM* and ensure that the *Allow* box of every item in *Permissions for SYSTEM* is checked (if it is checkable). Do the same check for *Administrators*.
+    4) Click on *OK* to go back to *Properties* dialog. Then click on *Advanced*.
+    5) Click on *Change Permissions*.
+    6) On *Permissions* tab, double click on *Administrators*.
+    7) Select *This folder, subfolders and files* for *Applies to* field and tick all the available *Basic permissions*. After that hit *OK*.
+    8) Do the same operation above (from item 7) for *SYSTEM*.
+    9) Click on *OK* all the way out. Try installing AdGuard again.
+
+### Error 1618: Another installation is already in progress {#error-1618}
+
+This error occurs when there are several instances of AdGuard VPN installer launched at the same time. What to do if you get this error:
+
+- Reboot your PC and start the installer again. When you restart the computer, all ongoing processes will stop, including all copies of the installer.
+
+- Don't make multiple clicks on the installer even if doesn't start right away. Sometimes it may take a few seconds to display the installer UI.
+
+### Error 1638: Another version of this product is already installed {#error-1638}
+
+It's very likely that you've already installed AdGuard VPN before.
+
+- Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
+
+- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+
+### Other errors {#other}
+
+If you've encountered an error that's not listed above, it is possible that we can solve it by ourselves. But in order to do that, we need log files from you. Please perform the following steps:
+
+- Find and archive **AdGuard VPN installation logs** in a same way as it is described in [this article](https://adguard.com/kb/adguard-for-windows/solving-problems/installation-logs/).
+
+- Find and save to disk **Event Viewer** logs. [This article](https://adguard.com/kb/adguard-for-windows/solving-problems/system-logs/) explains how to do that.
+
+- Please email all these files from two previous steps to the support team at **support@adguard.com** and describe the problem in the message body. Our tech support agents will reply to you as soon as possible.
```

Comment:
Please email all these files from the two previous steps at **support@adguard.com** and describe the problem in the message body. Our support team will reply to you as soon as possible.